backend/accounts.go

```diff
@@ -302,3 +302,25 @@ func (backend *Backend) SetTokenActive(accountCode string, tokenCode string, act
 		return acct.SetTokenActive(tokenCode, active)
 	})
 }
+
+// RenameAccount renames an account in the accounts database.
+func (backend *Backend) RenameAccount(accountCode string, name string) error {
+	if name == "" {
+		return errp.New("Name cannot be empty")
+	}
+	return backend.config.ModifyAccountsConfig(func(accountsConfig *config.AccountsConfig) error {
+		found := false
+		for i := range accountsConfig.Accounts {
```

Comment:
or, in fewer lines:

```go
for _, acct := range accountsConfig.Accounts {
  if acct.Code == accountCode {
    acct.Name = name
    return nil
  }
}
return errp.Newf("Account not found: %s", accountCode)
```

No need for `found` var.

Reply:
excellent!